drivers/usb/usbport/endpoint.c

```diff
@@ -1074,7 +1075,8 @@ USBPORT_OpenPipe(IN PDEVICE_OBJECT FdoDevice,
                 KeReleaseSpinLock(&Endpoint->EndpointSpinLock,
```

Comment:
Is there a purpose to release here then "immediately" reacquire this lock, or could that be optimized?